src/com/google/enterprise/adaptor/documentum/DocumentumAdaptor.java

```diff
@@ -588,6 +596,29 @@ private void validateDocumentTypes(IDfSession dmSession) {
     validatedDocumentTypes.addAllAbsent(validTypes);
   }
 
+  private void validateUpdatedDocumentsQuery(IDfSession session)
+      throws DfException {
+    Checkpoint checkpoint = Checkpoint.incremental();
+    String queryStr =
+        MessageFormat.format(updatedDocumentsQuery,
+            checkpoint.getLastModified(), checkpoint.getObjectId());
+    IDfQuery query = dmClientX.getQuery();
+    query.setDQL(queryStr);
+    IDfCollection result = null;
+    try {
+      result = query.execute(session, IDfQuery.DF_EXECREAD_QUERY);
+      result.next();
+    } catch (DfException e) {
+      logger.log(Level.WARNING,
+          "Error validating updatedDocumentsQuery {0}: {1}", new Object[] {
```

Comment:
Choose between English, "modified documents query", and the actual property name, "documentum.modifiedDocumentsQuery".